OIPA/api/activity/views.py

```diff
@@ -1,23 +1,37 @@
 from rest_framework import generics
 from iati.models import Activity
 from api.activity import serializers
-from api.generics.views import DynamicListAPIView
-from api.generics.views import DynamicRetrieveAPIView
 from api.generics.filters import BasicFilterBackend
 from api.generics.filters import SearchFilter
 from rest_framework.filters import OrderingFilter
 from api.activity import filters
+from api.activity.aggregation import AggregationsSerializer
 
 
-class ActivityList(DynamicListAPIView):
+class ActivityList(generics.ListAPIView):
```

Comment:
@vplme please import `ListAPIView` directly. It's confusing which `generics` module is used, since API both `rest_framework` and `api` have this module.